### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by RedTailedHawk** [antlr4ide/antlr4ide#108]
I'm getting the following errors when running "mvn clean package":

```
[INFO] --- xtend-maven-plugin:2.7.3:compile (default) @ antlr4ide.core ---
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
9: org.abego.treelayout.util.DefaultTreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
11: org.abego.treelayout.TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
106: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
164: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
261: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
221: DefaultTreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
256: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
241: DefaultTreeForTreeLayout cannot be resolved.

[INFO] Reactor Summary:
[INFO] 
[INFO] antlr4ide ......................................... SUCCESS [  0.037 s]
[INFO] ANTLR v4 IDE Runtime .............................. SUCCESS [  1.636 s]
[INFO] ANTLR v4 IDE Core ................................. FAILURE [ 25.559 s]
[INFO] ANTLR v4 IDE UI ................................... SKIPPED
[INFO] ANTLR v4 IDE Tests ................................ SKIPPED
[INFO] ANTLR v4 IDE Application .......................... SKIPPED
[INFO] ANTLR v4 IDE SDK .................................. SKIPPED
[INFO] ANTLR 4 IDE - Update Site ......................... SKIPPED
```

Please advise.

Thanks!
-Larry

**Comment by rashmir11** [comment]
I'm getting the same error. Has anyone found a solution for the above problem?

**Comment by ppsaltis** [comment]
I have the same problem.

**Comment by g4rr3t** [comment]
I have the same issue.

**Comment by arifogel** [comment]
Run it twice. It will work the second time. I'm still trying to figure out why this happens.

**Comment by timeraider4u** [comment]
Okay, I have been able to track down what causes the issue: The jar-file in the library sub-directory is no available when this project is first inspected by the maven system. So it does not get added to the class-path of the xtend-maven-plugin. This library is downloaded during module-execution (see the ant-maven run). It seems the xtend-maven plug-in does not recognize that the library is now existing (the classpath entries mentioned in the META-INF/Manifest.MF file are not re-checked for availability). And, as a result, the compilation of the xtend file fails. I have created an example project and reported the bug at https://github.com/eclipse/xtext-xtend/issues/72
